web/ui_utils.js

```diff
@@ -17,7 +17,8 @@
 'use strict';
 
 var CSS_UNITS = 96.0 / 72.0;
-var DEFAULT_SCALE = 'auto';
+var DEFAULT_SCALE_VALUE = 'auto';
+var DEFAULT_SCALE = 1.0;
```

Comment:
Changing these constants requires updating `web/viewer.js` too, because in its current form the default zoom value is now `100%` instead of `auto` in the standard viewer.